closure/private/defs.bzl

```diff
@@ -42,16 +43,20 @@ JS_HIDE_WARNING_ARGS = [
     "--hide_warnings_for=.soy.js",
```

Comment:
Can you remove ".soy.js" here? I think all .soy.js files live under bazel-out/local-fastbuild/genfiles/.